mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -425,7 +425,7 @@ The following arguments are supported:
 * `query` - (Required) A query that BigQuery executes when the view is referenced.
 
 * `use_legacy_sql` - (Optional) Specifies whether to use BigQuery's legacy SQL for this view.
-    The default value is true. If set to false, the view will use BigQuery's standard SQL.
+    If set to false, the view will use BigQuery's standard SQL.
```

Comment:
Could we add if unset behavior or a link to the api docs as above

Reply:
Yes, updated the description here.